main.go

```diff
@@ -45,7 +45,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
 	infrav1alpha4 "github.com/nutanix-cloud-native/cluster-api-provider-nutanix/api/v1alpha4"
-	infrav1beta1 "github.com/nutanix-cloud-native/cluster-api-provider-nutanix/api/v1beta1"
+	infrastructurev1beta1 "github.com/nutanix-cloud-native/cluster-api-provider-nutanix/api/v1beta1"
```

Comment:
I see that in other locations there is `infrav1` , `v1beta1` and `infrav1beta1`.
IMO `infrav1` makes it much simpler to refactor later when there is an API bump.

Reply:
In https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/pull/344/files#r1427199867, @deepakm-ntnx explained that `Kubebuilder now uses complete name of the package`. I agree with @dkoshkin though that keeping the shorter name and not the full API version (i.e. `infrav1`) is much easier for refactoring/updating in future. IMO kubebuilder conventions are a good start, but aren't always the best conventions to follow, this being one example of that.

Reply:
ok, will make it infrav1 here